pluma/pluma-window-activatable.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * pluma-window-activatable.h
+ * This file is part of pluma
+ *
+ * Copyright (C) 2010 Steve Frécinaux
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU Library General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Library General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Library General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
```

Comment:
from now on it's recommended to use the version that includes the URL instead of the posting address.

https://www.gnu.org/licenses/gpl-howto.html
> You should have received a copy of the GNU General Public License
> along with Foobar.  If not, see <https://www.gnu.org/licenses/>.

Foobar -> Pluma or Foobar `this program ` if you prefer